workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -17,64 +13,35 @@ public class Bootstrap : MonoBehaviour
 
         private const int TargetFrameRate = -1; // Turns off VSync
 
-        public const string LoggerName = "Bootstrap";
-
-        private static readonly List<WorkerBase> Workers = new List<WorkerBase>();
-
-        private static ConnectionConfig connectionConfig;
-
         public void Awake()
         {
-            InitializeWorkerTypes();
             // Taken from DefaultWorldInitalization.cs
             SetupInjectionHooks(); // Register hybrid injection hooks
             PlayerLoopManager.RegisterDomainUnload(DomainUnloadShutdown, 10000); // Clean up worlds and player loop
 
             Application.targetFrameRate = TargetFrameRate;
+            Worker.OnConnect += w => Debug.Log($"{w.WorkerId} is connecting");
+            Worker.OnDisconnect += w => Debug.Log($"{w.WorkerId} is disconnecting");
             if (Application.isEditor)
             {
-#if UNITY_EDITOR
-                var workerConfigurations =
-                    AssetDatabase.LoadAssetAtPath<ScriptableWorkerConfiguration>(ScriptableWorkerConfiguration
-                        .AssetPath);
-                foreach (var workerConfig in workerConfigurations.WorkerConfigurations)
+                var config = new ReceptionistConfig
                 {
-                    if (!workerConfig.IsEnabled)
-                    {
-                        continue;
-                    }
-
-                    var worker = WorkerRegistry.CreateWorker(workerConfig.Type, $"{workerConfig.Type}-{Guid.NewGuid()}",
-                        workerConfig.Origin);
-                    Workers.Add(worker);
-                }
-
-                connectionConfig = new ReceptionistConfig();
-                connectionConfig.UseExternalIp = workerConfigurations.UseExternalIp;
-#endif
+                    WorkerType = "UnityClient",
+                };
+                CreateWorker(config, Vector3.zero);
+                config = new ReceptionistConfig
+                {
+                    WorkerType = "UnityGameLogic",
+                };
+                CreateWorker(config, new Vector3(10, 0, 0));
```

Comment:
Inline the creation of the configs here -

`CreateWorker(new ReceptionistConfig("UnityClient"), Vector3.zero)`